### PR TITLE
Ops handbook finance management

### PR DESCRIPTION
**Pull request by demkodo**
Big PR and chances of an unhappy Travis - let' see :)

-----
[View rendered activities/finance-management/authorization.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/finance-management/authorization.md)
[View rendered activities/finance-management/bank.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/finance-management/bank.md)
[View rendered activities/finance-management/bookkeeping.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/finance-management/bookkeeping.md)
[View rendered activities/finance-management/index.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/finance-management/index.md)
[View rendered activities/finance-management/insurances.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/finance-management/insurances.md)
[View rendered activities/finance-management/reporting.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/finance-management/reporting.md)
[View rendered activities/office-management/index.md](https://github.com/publiccodenet/about/blob/ops-handbook-finance-management/activities/office-management/index.md)

### CODE REVIEW

File: activities/finance-management/bookkeeping.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+type: Guide
+explains: how bookkeeping is managed
+---
+
+# Bookkeeping
+
+[GetMonkeys](https://getmonkeys.io/) is our service provider. They process and book all the payments every second Wednesday of the month. Any corrections can be mailed to the bookkeeper and are processed the next day. GetMonkey's is also responsible for producing our annual financial reports and can be asked for other financial services based on our needs.
```
Comment (ElenaFdR) [inline activities/finance-management/bookkeeping.md:8]:
```suggestion
[GetMonkeys](https://getmonkeys.io/) is our service provider. They process and book all the payments every second Wednesday of the month. Any corrections can be mailed to the bookkeeper and are processed the next day. GetMonkeys is also responsible for producing our annual financial reports and can be asked for other financial services based on our needs.
```

File: activities/finance-management/bookkeeping.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Exact
+
+[Exact is a Dutch software company](https://www.exact.com/) that offers accounting, ERP, and other software for small and medium enterprises. We have a license via GetMonkey’s. The Operations Coordinator and Chief Executive have access to Exact. Temporary access can be given to financial consultants.
```
Comment (ElenaFdR) [inline activities/finance-management/bookkeeping.md:12]:
```suggestion
[Exact is a Dutch software company](https://www.exact.com/) that offers accounting, ERP, and other software for small and medium enterprises. We have a license via GetMonkeys. The Operations Coordinator and Chief Executive have access to Exact. Temporary access can be given to financial consultants.
```

File: activities/finance-management/reporting.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+A concise financial one-pager is produced by the Operations Coordinator for the bi-monthly board meeting. The one-pager gives an overview of:
+
+* The current cash position
+* Costs incurred
+* Runway
+* General information (depending on time of the year, e.g. updates on (new) funding sources, important budget changes and interim conclusion)
+
+## Annual reporting
+
+Annual reporting consists of the annual financial statement and an annual report. The latter is a powerful piece of communications that shows how we deliver on our mission, and celebrate our accomplishments.
```
Comment (lscheske) [inline activities/finance-management/reporting.md:19]:
```suggestion
As an ANBI, we are legally obliged to publish two kinds of reports: an annual financial statement (Jaarrekening, see below) and an annual report (Jaarverslag). The latter is a powerful piece of communications that shows how we deliver on our mission, achieve the goals set out in our policy plan, and celebrate our accomplishments.
```

File: activities/finance-management/reporting.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Financial annual statement
```
Comment (lscheske) [inline activities/finance-management/reporting.md:21]:
```suggestion
## Annual financial statement
```

File: activities/finance-management/reporting.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Our first financial year runs from 1 July 2019 - 31 December 2020. The Operations Coordinator is working closely with the bookkeeper to produce the statement. An ANBI has to publish the financial statement on our website. The financial statement may concern the following information:
```
Comment (lscheske) [inline activities/finance-management/reporting.md:23]:
```suggestion
The Operations Coordinator is responsible for producing the annual financial statement (Jaarverslag), in close collaboration with our bookkeeper. The financial statement should include the following information:
```

File: activities/finance-management/reporting.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+* The balance sheet
+* The profit and loss statement
+* An additional explanation
+
+The financial statement must be published on our website within 6 six months after the end of the financial year.
```
Comment (lscheske) [inline activities/finance-management/reporting.md:29]:
```suggestion
The financial statement must be published on our website within six months after the end of the financial year.
```

File: activities/finance-management/reporting.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+[Template for our annual financial statement.](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_)
+
+## King Baudouin Foundation United States (KBFUS)
+
+The King Baudouin Foundation’s mission is to contribute to a better society. The Foundation is an actor for change and innovation, serving the public interest and increasing social cohesion in Belgium and Europe. They seek to maximize their impact by strengthening the capacity of organizations and individuals. They also stimulate effective philanthropy by individuals and corporations. The Foundation’s key values are integrity, transparency, pluralism, independence, respect for diversity, and promoting solidarity. The Foundation’s current areas of activity are social justice & poverty, health, heritage and culture, social engagement, Africa, Latin America, Asia, Education and development of talents, Europe, climate, the environment and biodiversity. The King Baudouin Foundation is a public benefit foundation.
```
Comment (lscheske) [inline activities/finance-management/reporting.md:35]:
```suggestion
```

Comment (lscheske) [reply activities/finance-management/reporting.md:35]: No need to include this information about them - this section is about reporting and what we need to deliver based on our legal agreements with them. Providing the link to the drive with all the background info should be sufficient.

**Review by lscheske — COMMENTED** [review]
I've made a couple of editorial suggestions throughout.

File #11 is the same as PR #831 and also duplicates a change made in PR #832

File: activities/finance-management/bookkeeping.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+## Budget
+
+* The budget blueprint (spreadsheet) which is used as the baseline for budgeting and forecasting was constructed with help of our finance consultant, Rob Schouten.
+* The budget is approved by the board by the end of each year and is added to Visionplanner
+* The Operations Coordinator is responsible for budgeting and adding it to Visionplanner
+
+## Tax status pertaining financials
+
+* We have ANBI status: read more information about ANBI benefits and requirements
```
Comment (lscheske) [inline activities/finance-management/bookkeeping.md:54]: Link is missing to the "Read more information about ANBI benefits and requirements"

**Review by lscheske — COMMENTED** [review]
Just some minor changes, then this is good to go.

File: activities/finance-management/authorization.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+---
+type: Guide
+explains: how authorization is managed at the Foundation
+---
+
+# Authorization
+
+Board members – two board members can decide on payments above € 50.000. [See bylaws.](../../organization/articles-of-association.md)
```
Comment (ElenaFdR) [inline activities/finance-management/authorization.md:8]:
```suggestion
Two board members can approve payments above € 50.000. [See bylaws.](../../organization/articles-of-association.md)
```

Comment (ElenaFdR) [inline activities/finance-management/authorization.md:8]: In American/British English, you don't usually put a space between a currency symbol and currency amount.

File: activities/finance-management/bank.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+---
+type: Guide
+explains: how banking is managed at the Foundation
+---
+
+# Bank
+
+Our bank is the Rabobank. There is a checking and savings account. We are in progress of opening a USD account. There are bank cards and credit cards and online keys. Bank cards:
```
Comment (ElenaFdR) [inline activities/finance-management/bank.md:8]:
```suggestion
Our bank is the Rabobank. There is a EUR checking and savings account. We are in progress of opening a USD account. There are bank cards and credit cards and online keys. Bank cards:
```

File: activities/finance-management/reporting.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
+---
+type: Guide
+explains: reporting requirements
+---
+
+# Reporting
+
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:7]: Suggestion: add an intro sentence for context.

```suggestion

We produce financial reports for different audiences throughout the year.
```
```

File: activities/finance-management/reporting.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+* Runway
+* General information (depending on time of the year, e.g. updates on (new) funding sources, important budget changes and interim conclusion)
+
+## Annual reporting
+
+As an ANBI, we are legally obliged to publish two kinds of reports: an annual financial statement (Jaarrekening, see below) and an annual report (Jaarverslag). The latter is a powerful piece of communications that shows how we deliver on our mission, achieve the goals set out in our policy plan, and celebrate our accomplishments.
+
+## Annual financial statement
+
+The Operations Coordinator is responsible for producing the annual financial statement (Jaarverslag), in close collaboration with our bookkeeper. The financial statement should include the following information:
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:23]: Are the contents of the jaarverslag optional?
```suggestion
The Operations Coordinator is responsible for producing the annual financial statement (jaarverslag), in close collaboration with our bookkeeper. The financial statement must include the following information:
```

File: activities/finance-management/reporting.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+* The balance sheet
+* The profit and loss statement
+* An additional explanation
+
+The financial statement must be published on our website within six months after the end of the financial year.
+
+[Template for our annual financial statement.](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_)
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:31]:
```suggestion
[Template for our annual financial statement](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_).
```

File: activities/finance-management/reporting.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+## King Baudouin Foundation United States (KBFUS)
+
+We opened a fund at the KBFUS Foundation to enable smooth faciliation of receiving gifts from US donors. The name of our fund is [“Foundation for Public Code US Fund”.](https://kbfus.networkforgood.com/projects/52915-p-kbfus-funds-foundation-for-public-code-nl)
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:35]: Is the word 'foundation' redundant in 'KBFUS Foundation'?

Comment (ElenaFdR) [inline activities/finance-management/reporting.md:35]:
```suggestion
We opened a fund at the KBFUS Foundation to facilite receiving gifts from US donors. The name of our fund is [“Foundation for Public Code US Fund”](https://kbfus.networkforgood.com/projects/52915-p-kbfus-funds-foundation-for-public-code-nl).
```

File: activities/finance-management/reporting.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+
+The Operations Coordinator has log-in credentials to the grants interface where the fund is managed and reports can be filed. [More information about the fund and how the process works can be found on the Drive.](https://drive.google.com/drive/folders/1hv1XKbA6y8XYO9KqbKo5-TFlFxRROn0R)
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:37]:
```suggestion
The Operations Coordinator has log-in credentials to the grants interface where the fund is managed and reports can be filed. [More information about the fund and how the process works](https://drive.google.com/drive/folders/1hv1XKbA6y8XYO9KqbKo5-TFlFxRROn0R) can be found on the Drive.
```

File: activities/finance-management/reporting.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+
+### KBFUS annual report
+
+The Foundation for Public Code needs to provide a narrative and financial report on the use of grant funds to KBFUS no less than annually and final reports upon completion of the grant period and until grant funds are expended in full. The reports should include a narrative account of how grant funds were used, what was accomplished by the expenditure of funds (including a description of progress made towards achieving the goals of the grant), and compliance with the terms of the Agreement, along with a financial statementand copies of any publications resulting from the grant.
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:41]:
```suggestion
The Foundation for Public Code needs to provide a narrative and financial report on the use of grant funds to KBFUS no less than annually and final reports upon completion of the grant period and until grant funds are expended in full. The reports should include a narrative account of how grant funds were used, what was accomplished by the expenditure of funds (including a description of progress made towards achieving the goals of the grant), and compliance with the terms of the Agreement, along with a financial statement and copies of any publications resulting from the grant.
```

Comment (ElenaFdR) [reply activities/finance-management/reporting.md:41]: Does our agreement with them have any other description than 'Agreement'?

File: activities/finance-management/reporting.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+The report should be signed by an authorized officer, and the financial statement should be attested by the responsible financial officer or a certified public accountant, and should reflect the categories stipulated in the attached approved budget. The report shall include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received. [Detailed financial requirements can be found on our Drive.](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj)
```
Comment (ElenaFdR) [inline activities/finance-management/reporting.md:43]:
```suggestion
The report should be signed by an authorized officer, and the financial statement should be attested by the responsible financial officer or a certified public accountant, and should reflect the categories stipulated in the attached approved budget. The report must include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received. [Detailed financial requirements can be found on our Drive](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj).
```

File: activities/finance-management/authorization.md [hunk 18]
```diff
@@ -0,0 +1,12 @@
+---
+type: Guide
+explains: how authorization is managed at the Foundation
+---
+
+# Authorization
+
+Two board members can approve payments above € 50.000. [See bylaws.](../../organization/articles-of-association.md)
```
Comment (ElenaFdR) [inline activities/finance-management/authorization.md:8]: Suggest changing all periods in numbers in the thousands to commas, as is more common in English.

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
Very clear and complete explanation of our financial administration!
Some small proofreading suggestions, but otherwise very close to being ready to go.

**Review by ElenaFdR — DISMISSED** [review]
Amazing work!

File: activities/finance-management/index.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
+---
+type: Index
+---
+
+# Finance management
+
+Our financial management includes planning, organizing, directing and controlling the financial activities of the association. It includes budget management, financial administration, bookkeeping, and reporting in compliance with our status as a public benefit organization (ANBI).
+
+* [Authorization](authorization.md)
```
Comment (ElenaFdR) [inline activities/finance-management/index.md:9]:
```suggestion
* [Financial authorization](authorization.md)
```

**Review by ElenaFdR — APPROVED** [review]

**Comment by clausmullie** [comment]
In the interest of iterative working, I'm going to merge this. Awesome stuff! :+1:

**Pull request merged**